app/kubernetes-ingress-controller/1.3.x/concepts/deployment.md

```diff
@@ -103,7 +103,7 @@ has the above permissions. The Ingress Controller Pod then has this
 necessary authentication and authorization tokens to communicate with the
 Kubernetes API-server.
 
-[rbac.yaml](https://github.com/Kong/kubernetes-ingress-controller/blob/main/deploy/manifests/base/rbac.yaml) contains the permissions
+[rbac.yaml](https://github.com/Kong/kubernetes-ingress-controller/blob/main/config/base/rbac.yaml) contains the permissions
```

Comment:
```suggestion
[rbac.yaml](https://github.com/Kong/kubernetes-ingress-controller/tree/main/config/rbac) contains the permissions
```
This link isn't working; I went looking for the new location, and it looks like there's way more that just one RBAC file now, so I linked it to the folder instead.